src/types/models/ExchangeRate.ts

```diff
@@ -0,0 +1,51 @@
+// Auto-generated , DO NOT EDIT
+import {Entity} from "@subql/types";
+import assert from 'assert';
+
+import {
+    ExchangeData,
+} from '../interfaces'
+
+
+export class ExchangeRate implements Entity {
+
+    constructor(id: string) {
+        this.id = id;
+    }
+
```

Comment:
maybe you can add prettier to format code, you can refer to https://github.com/parallel-finance/stake-client